pupa/utils/model_ops.py

```diff
@@ -0,0 +1,113 @@
+from django.db import transaction
+from django.db.models import get_models, Model
+from django.contrib.contenttypes.generic import GenericForeignKey
+
+import json
+import logging
+
+logger = logging.getLogger("")
+
+@transaction.commit_on_success
+def merge_model_objects(primary_object, alias_objects=[], keep_old=False):
```

Comment:
this code mirrors some of what I did in python-opencivicdata-django. I think we need to resolve the following questions:

1) Would we prefer to have this code in pupa or python-ocd-django?

2) Do we want more generic code that can merge any two instances of the same class, or custom code that has specific checks and does specific things for each model? Or do we want both to be available?

I'm going to PR my code shortly and will link to it here as we think about this.